packages/api-reference/src/components/Content/Content.vue

```diff
@@ -46,6 +46,14 @@ const fallBackServer = useRefOnMount(() => {
 const localServers = computed(() => {
   if (props.parsedSpec.servers && props.parsedSpec.servers.length > 0) {
     return props.parsedSpec.servers
+  } else if (
+    props.parsedSpec.host &&
+    props.parsedSpec.schemes &&
+    props.parsedSpec.schemes.length > 0
```

Comment:
could this change to

```
props.parsedSpec.schemes?.length > 0
```

Reply:
We could replace it with just `props.parsedSpec.schemes?.length`, but not with `props.parsedSpec.schemes?.length > 0`, because `undefined.length` can’t return a number. 🤡 

I tried to keep it similar to the lines before, where we have a `…length > 0`. Anyway, whatever you think is right.

Reply:
ah good catch, i forgot about that!